frontend/sozisel-app/src/components/LoginScreen/LoginScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import "./LoginScreen.scss";
+
+import * as yup from "yup";
+
+import Button from "../utils/Button/Button";
+import Card from "../utils/Card/Card";
+import ErrorMessage from "../utils/Input/ErrorMessage";
+import Input from "../utils/Input/Input";
+import Navbar from "../Navbar/Navbar";
+import { ReactElement } from "react";
+import Spinner from "../utils/Spinner/Spinner";
+import conference_img from "../../assets/conference_img.png";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router";
+import { useLoginMutation } from "../../graphql";
+import { useTranslation } from "react-i18next";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+interface LoginFormData {
+  email: string;
+  password: string;
+}
+
+const loginSchema = (invalidEmailFormat: string, fieldRequired: string) => {
```

Comment:
can't that be just an `yup.object().shape...`?
What I meant by passing error strings was osmething like this 
```js
const schema = yup.object().shape({
    email: yup.string().email("errorMessages.invalidEmailFormat").required("errorMessages.fieldRequired"),
    password: yup.string().required("errorMessages.fieldRequired"),
  });
```
And then in form errors you do something like this 
```js
error={errors.email && t(errors.email)}
```

And on the top of that change `errorMessages` to `inputErrors` or something. We already know that it is a mesasge.